src/styles/issues.module.scss

```diff
@@ -43,10 +46,22 @@ $border-color: #d9d9d9;
     padding: 0.25rem 0.75rem;
     font-size: 0.875rem;
     border-radius: 0.25rem;
-    border: 1px solid $border-color;
+    border: none;
+    background-color: $color-disabled;
+
+    &:focus {
+        outline: none;
+    }
```

Comment:
setting outline none to `none`  may not give feedback for users on using keyboard to focus different button.

Reply:
removed